src/glmfit.jl

```diff
@@ -115,7 +115,23 @@ function confint(obj::AbstractGLM, level::Real)
 end
 confint(obj::AbstractGLM) = confint(obj, 0.95)
 
-deviance(m::AbstractGLM)  = deviance(m.rr)
+deviance(m::AbstractGLM) = deviance(m.rr)
+
+function loglikelihood(m::AbstractGLM)
```

Comment:
@nalimilan I stumbled over this  thread and remember an old R-help conversation, maybe you find this helpful:
http://r.789695.n4.nabble.com/GLM-output-for-deviance-and-loglikelihood-tt3451911.html#none